src/tracing.ts

```diff
@@ -0,0 +1,49 @@
+import { SpanStatusCode } from "@opentelemetry/api"
+import { addSpanAttributes, currentSpan } from "./opentelemetry/api"
+
+export const enum BundleActionStatus {
+    SKIPPED = 'SKIPPED',
+    SUCCESS = 'SUCCESS',
+    FAILED = 'FAILED'
+}
+
+
+export const addBundleId = (bundleId: string) => {
+    addSpanAttributes({
+        'orchestrator.bundle.id': bundleId,
+    })
+}
+
+export const addFillStatus = (status: BundleActionStatus) => {
+    addSpanAttributes({
+        'fill.status': status.toString(),
+    })
+}
+
+export const addClaimStatus = (status: BundleActionStatus) => {
+    addSpanAttributes({
+        'claim.status': status.toString(),
```

Comment:
Thoughts on adding optional tx hash field in here? May be nice to be able to lookup by hash

Reply:
Check addTransactionId calls. You don't always have transaction id on failed claims/fills (i.e. RPC call failed because of network)

Reply:
Missed `addTransactionId`. Resolving